### PR TITLE
Change invalid parameter in docstring

### PR DESCRIPTION
**Pull request by nikssardana**
Renamed result_list to result in docstring for print_affected_files.
Changed content of the docstring for result parameter.

Closes https://github.com/coala/coala/issues/2809

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
hey, the commit message is not quite according to our standards.

Hi,

it seems that your commit message doesn't contain enough information. Don't worry, most people don't know about good commit messages when they join us, including very experienced developers :) Standards vary from project to project.

The most important part is the first line of your commit message - maintainers will use it to read through a large number of commits and thus it's important that it's short but most importantly concise. I'd heartily recommend reading http://coala.io/commit !

To change your message you just use `git commit --amend` and then `git push --force` the new commit to replace the old one.

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix up the message for you - just tell us :).

Cheers!

**Comment by nikssardana** [comment]
@Adrianzatreanu I'll go throughthe guide and push the patch again after making the changes. Thanks! I'd really want to work on this project for a long term. Cheers! :smile:

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -321,7 +321,7 @@ def print_affected_files(console_printer,
     :param console_printer: Object to print messages on the console.
     :param log_printer:     Printer responsible for logging the messages.
     :param section:         The section to which the results belong to.
-    :param result_list:     List containing the results
+    :param result:          The actual result
```
Comment (sils1297) [inline coalib/output/ConsoleInteraction.py]: other params have a period, please go with the surrounding style

Comment (sils1297) [inline coalib/output/ConsoleInteraction.py]: also the description could be better: `The result to print the context for.`?

File: coalib/output/ConsoleInteraction.py [hunk 2]
```diff
@@ -321,7 +321,7 @@ def print_affected_files(console_printer,
     :param console_printer: Object to print messages on the console.
```
Comment (sils1297) [inline coalib/output/ConsoleInteraction.py]: can you file a newcomer issue for line 319 and it's typos? (check if it already exists though!)

**Comment by nikssardana** [comment]
@sils1297 sure I'll add the period.
And yes I'll look into that line and add a newcomer issue too.

**Comment by nikssardana** [comment]
And yes,the description you suggested sounds better. Will change that too.

File: coalib/output/ConsoleInteraction.py (hunk 2 above)
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: it seems the commit message is still not perfect. first of all, youre not mentioning where that docstring is. it should at least be something like: ``ConsoleInteraction: ..`` or ``Console:`` 

also, in commit body, dont use past tense, use present simple, as in: "Changes".

**Comment by Adrianzatreanu** [comment]
as gitmate states, your commit body has a few lines that are over 72 characters, please wrap them up using newlines :D thank you

File: coalib/output/ConsoleInteraction.py (hunk 2 above)
Comment (nikssardana) [inline coalib/output/ConsoleInteraction.py]: Okay @Adrianzatreanu . I took care of that. Silly me :D

**Comment by nikssardana** [comment]
Thanks @Adrianzatreanu . The circleci tests fail too often for me :disappointed: 
But i'll learn with time :smile:

File: coalib/output/ConsoleInteraction.py (hunk 2 above)
Comment (nikssardana) [inline coalib/output/ConsoleInteraction.py]: @Adrianzatreanu i tried it earlier, but the message was too long. Couldn't come up with a shorter message with ConsoleInteraction.

Comment (nikssardana) [inline coalib/output/ConsoleInteraction.py]: I changed the message now.

**Comment by Adrianzatreanu** [comment]
ack bd0c85d7a5e196af5a619e7beb278587b9911383

**Comment by Adrianzatreanu** [comment]
@rultro merge

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**